### ISSUE TITLE
disableBodyScroll causing issue with position: fixed elements

### ISSUE THREAD

**Issue by anishbenji** [angular/material#5915]
disableBodyScroll from $mdUtil causes position: fixed elements to offset by scroll amount when disableBodyScroll is invoked.

**Comment by DevVersion** [comment]
It would be great if you provide a codepen.. then we can investigate better.
I already tried to reproduce your issue, but I didn't get it.

**Comment by anishbenji** [comment]
This is embarrassing. It turns out I needed to set the `top` and `left` styles for the fixed for the `position: fixed` element. Now it works fine. I've included the following Codepen just for reference.

http://codepen.io/anishbenji/pen/YymqOL

Just scroll down the page some amount and click the 'Alert Dialog' button on the toolbar. The toolbar will move upward by the scroll amount, but the FAB button stays put as expected.
The toolbar doesn't have `top` & `left` set in CSS.

Made from the Angular Material Demos:

1. I took an example from toolbar demo, modified it so that the toolbar has style of `position: fixed` and `md-content` scrolls with `body`. 
2. I merged one of the examples from the dialog demo.
3. I added a FAB button from the button demo.

Regards,
Anish
